tests/Containerfile

```diff
@@ -8,7 +8,7 @@ RUN python3 -m venv $VIRTUAL_ENV
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 
 RUN pip install pipenv
-COPY Pipfile Pipfile
```

Comment:
Is it correct to remove the pipfile here? Not a python expert, but I'd expect both files are needed.

Reply:
The Pipefile is used to generate the `Pipfile.lock`. 

The generation of the Pipfile.lock is moved outside of the container, thus `pipenv` must not re-generate a different `Pipfile.lock` during the container build step. 
Therefore I removed the Pipfile, so pipenv does not even try to generate it. 

However, we could add a `--ignore-pipfile` to pipenv.

Reply:
Hm.. let me check. if we even have to use `--ignore-pipfile`.

Reply:
Ok, it's either `--ignore-pipfile` or adding the `Pipfile`to the container image. 

Added Pipfile to the container image now